src/components/claiming/WalletDisclaimer.vue

```diff
@@ -40,13 +40,15 @@ export default {
   data() {
     return {
       texts: {},
+      disclaimer_2: 'Important',
     }
   },
   methods: {
     previousStep() {
       this.$router.push('/claiming/create-wallet')
     },
     nextStep() {
+      this.$store.commit('addDisclaimer', { key: this.disclaimer_2 })
```

Comment:
We don't need to add disclaimers in each step. I think we can assume that the user has accept of the disclaimers if they have create the wallet.